test/common/stats/thread_local_store_test.cc

```diff
@@ -72,6 +72,13 @@ class HistogramWrapper {
   histogram_t* histogram_;
 };
 
+namespace {
+void expectStatsEq(const Stats::HistogramStatistics& lhs, const Stats::HistogramStatistics& rhs) {
+  EXPECT_EQ(lhs.supportedBuckets(), rhs.supportedBuckets());
```

Comment:
Tiny nit, but It's usually a little better to put the EXPECT calls directly in the test body so that an error points you unambiguously to the test & line.

however I also  love factoring stuff out, which your design here does nicely.

As a compromise, you could pass in a string annotation to this function and use different args at the call-sites to disambiguate should there be a failure. Then you just write:

 EXPECT_EQ(....) << annotation;

Reply:
I should probably just define `operator ==` for `HistogramStatistics`.